submissions/Valkirin/pop-up/index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Pup-up without JS</title>
+	<link rel="stylesheet" href="source.css">
+</head>
+<body>
+	<header class="header">
+		<nav class="nav">
+			<ul class="menu">
+				<li class="nav_item">
+	                <a href="#" class="link">Green</a>   
+	            </li>
+	            <li class="nav_item">
+	                <a href="#" class="link">Peace</a>
+	            </li>
+	            <li class="nav_item">
+					<input type="checkbox" id="popup" class="popUpControl">
+					<label for="popup" class="button">
```

Comment:
Click on the body of the pop-up closes it. It's happening 'cause you wrapped whole pop-up in `label`. Fix it so pop-up will show and hide only when user clicked on `.button` clicked.

Reply:
Thx for review. I'm solved it.